kpops/components/base_components/cleaner.py

```diff
@@ -20,6 +20,10 @@ class Cleaner(HelmApp, ABC):
     def full_name(self) -> str:
         return super().full_name + self.suffix
 
+    @property
+    def app_full_name(self) -> str:
+        return super().full_name
+
```

Comment:
A hack for now to have the full_name of the app without the `-clean` suffix

Reply:
could you elaborate?

Reply:
`super().full_name` will evoke from the `Cleaner` class. So, we will still have the suffix 
```python
    @property
    @override
    def full_name(self) -> str:
        return super().full_name + self.suffix
```

Reply:
that sounds like the behavior of `self.full_name`. I believe `super().full_name` does what you want (same as your new property without defining a function for it)

Reply:
how about `super(HelmApp).full_name`?

Reply:
```
AttributeError: 'super' object has no attribute 'full_name'
```

Reply:
my bad, needs to be `super(HelmApp, self).full_name`

Reply:
Cool! Done